incubator/catalog-schema-registry/src/main/java/io/aklivity/zilla/runtime/catalog/schema/registry/internal/SchemaRegistryCatalogHandler.java

```diff
@@ -58,8 +59,9 @@ public SchemaRegistryCatalogHandler(
         this.client = HttpClient.newHttpClient();
         this.request = new RegisterSchemaRequest();
         this.crc32c = new CRC32C();
-        this.cache = new Int2ObjectCache<>(1, 1024, i -> {});
-        this.schemaIdCache = new Int2ObjectCache<>(1, 1024, i -> {});
+        this.schemas = new Int2ObjectCache<>(1, 1024, i -> {});
+        this.schemaIds = new Int2ObjectCache<>(1, 1024, i -> {});
+        this.cacheTtl = config.cacheTtl * 1000;
```

Comment:
Let's rename this field to `maxAgeMillis` and then it will be `config.maxAge.toMillis()` as we are parsing `max-age` as a `Duration`.